pkg/features/kube_features.go

```diff
@@ -614,7 +615,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	LocalStorageCapacityIsolationFSQuotaMonitoring: {Default: false, PreRelease: featuregate.Alpha},
 	NonPreemptingPriority:                          {Default: false, PreRelease: featuregate.Alpha},
 	VolumePVCDataSource:                            {Default: true, PreRelease: featuregate.Beta},
-	PodOverhead:                                    {Default: false, PreRelease: featuregate.Alpha},
+	PodOverhead:                                    {Default: true, PreRelease: featuregate.Beta},
```

Comment:
Another place needs to be updated as well:

```
	// owner: @egernst
	// alpha: v1.16
	// <-- add a line indicating it's beta in 1.18
	// Enables PodOverhead, for accounting pod overheads which are specific to a given RuntimeClass
	PodOverhead featuregate.Feature = "PodOverhead"
```

Reply:
Can you clarify?I *think* this is already there?

Reply:
nvm... Not sure whey I didn't see that...